homeassistant/components/unifi/entity.py

```diff
@@ -83,6 +83,7 @@ async def async_added_to_hass(self) -> None:
         self.async_on_remove(
             handler.subscribe(
                 self.async_signalling_callback,
+                id_filter=self._obj_id,
             )
         )
```

Comment:
Should this be a separate PR for beta?

Reply:
I created one as well. https://github.com/home-assistant/core/pull/84787

Maybe should be merged first?

Reply:
yes